mysql.sh

```diff
@@ -0,0 +1,150 @@
+    
+    if container_running; then
+        print_warn "MySQL container '${CONTAINER_NAME}' is already running."
+        return 0
+    fi
+    
+    if container_exists; then
+        print_info "Starting existing MySQL container '${CONTAINER_NAME}'..."
+        docker start "${CONTAINER_NAME}"
+    else
+        print_info "Creating and starting new MySQL container '${CONTAINER_NAME}'..."
+        docker run -d \
+            --name "${CONTAINER_NAME}" \
+            -p "${DB_PORT}:3306" \
+            -e MYSQL_DATABASE="${DB_NAME}" \
+            -e MYSQL_USER="${DB_USER}" \
+            -e MYSQL_PASSWORD="${DB_PASSWORD}" \
+            -e MYSQL_ROOT_PASSWORD="${DB_ROOT_PASSWORD}" \
+            -v "${VOLUME_NAME}:/var/lib/mysql" \
+            "mysql:${MYSQL_VERSION}" \
+            --character-set-server=utf8mb4 \
+            --collation-server=utf8mb4_unicode_ci
+    fi
+    
+    # Wait for MySQL to be ready
+    wait_for_container "MySQL" \
+        "docker exec ${CONTAINER_NAME} mysqladmin ping -h localhost -u root -p${DB_ROOT_PASSWORD}"
+    
+    # Print connection details
+    print_connection_details \
+        "Host: localhost" \
+        "Port: ${DB_PORT}" \
+        "Database: ${DB_NAME}" \
+        "User: ${DB_USER}" \
+        "Password: ${DB_PASSWORD}" \
+        "Root Password: ${DB_ROOT_PASSWORD}" \
+        "JDBC URL: jdbc:mysql://localhost:${DB_PORT}/${DB_NAME}"
+}
+
+# Function to stop MySQL
+stop_mysql() {
+    stop_container "MySQL"
+}
+
+# Function to restart MySQL
+restart_mysql() {
+    stop_mysql
+    start_mysql
+}
+
+# Function to show status
+show_status() {
+    show_container_status "MySQL"
+}
+
+# Function to clean up (remove container and volume)
+clean_mysql() {
+    clean_container "MySQL"
+}
+
+# Function to show logs
+show_logs() {
+    show_container_logs "MySQL"
+}
+
+# Function to show help
+show_help() {
+    echo "MySQL Database Script for Nuxeo Development"
+    echo ""
+    echo "Usage: $0 {start|stop|restart|status|clean|logs|help}"
+    echo ""
+    echo "Commands:"
+    echo "  start    - Start MySQL container"
+    echo "  stop     - Stop MySQL container"
+    echo "  restart  - Restart MySQL container"
+    echo "  status   - Show container status"
+    echo "  clean    - Remove container and data volume"
+    echo "  logs     - Show container logs (follow mode)"
+    echo "  help     - Show this help message"
+    echo ""
+    echo "Environment Variables:"
+    echo "  DB_CONTAINER_NAME  - Container name (default: nuxeo-mysql-dev)"
+    echo "  DB_PORT            - Host port (default: 3306)"
+    echo "  DB_NAME            - Database name (default: nuxeo)"
+    echo "  DB_USER            - Database user (default: nuxeo)"
+    echo "  DB_PASSWORD        - Database password (default: nuxeo)"
+    echo "  DB_ROOT_PASSWORD   - Root password (default: root)"
+    echo "  MYSQL_VERSION      - MySQL version (default: 8.0)"
+    echo "  VOLUME_NAME        - Volume name (default: nuxeo-mysql-data)"
```

Comment:
Could you turn the environment variables for configuration to script options?

Same for other scripts.